benchmarks/Asset/MyConfiguration.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/*
+ * This file is part of the Disco package.
+ *
+ * (c) bitExpert AG
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace bitExpert\Disco\Asset;
+
+use bitExpert\Disco\Annotations\Configuration;
+use bitExpert\Disco\Annotations\Bean;
+
+/**
+ * @Configuration
+ */
+class MyConfiguration
```

Comment:
Better rename the class to BenchmarkConfiguration so that it is clear what the purpose of the class is.